media/js/privacy/privacy-quantum.js

```diff
@@ -1,37 +1,13 @@
 (function() {
     'use strict';
 
-    var mainContent = document.querySelector('main');
+    var mainContent = document.querySelector('.privacy-body');
```

Comment:
Hmm, we should probably also move these DOM queries to within a basic JS feature detect?